pages/about.html

```diff
@@ -4,9 +4,61 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About | Uyen Vong</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/about.css">
 </head>
 <body>
-  
+  <header>
+    <h1>about</h1>
+  </header>
+  <nav>
```

Comment:
Nice! Yes, we commonly see navbars implemented with an unordered list within a `<nav>` element. 👍🏻